src/main/java/org/eclipse/tractusx/ssi/lib/model/verifiable/credential/VerifiableCredentialBuilder.java

```diff
@@ -178,6 +179,9 @@ public VerifiableCredential build() {
     if (expirationDate != null) {
       map.put(VerifiableCredential.EXPIRATION_DATE, formatter.format(expirationDate));
     }
+    if (!Objects.isNull(credentialStatus)) {
```

Comment:
The intended scope of `Objects.isNull` is in a lambda expression. This can be replaced with `credentialStatus == null`.

Reply:
fixed.